makroer/panelfig.sas

```diff
@@ -0,0 +1,62 @@
+%macro panelfig;
+
+data norge;
+  set &tema.&teknikk._dp_tot_bohf(keep=rate20: bohf);
+  where bohf=8888;
+run;
+
+%macro trans_N(yr=);
+data yr&yr(keep=aar Nrate);
+  set norge;
+  aar=&yr;
+  rename rate&yr=Nrate;
+run;
+%mend;
+%trans_N(yr=2014);
+%trans_N(yr=2015);
+%trans_N(yr=2016);
+data N;
+  set yr2014 yr2015 yr2016;

```

Comment:
Gjøre denne også generell, slik som for merge-makroen?

Reply:
Ja, det bør vi absolutt. Det er vel den tidstrend-figuren som skal ha 5 år? Så det må jo også endres mer på den, men det kan vel gjøres etterhvert.